src/SparkFun_u-blox_SARA-R5_Arduino_Library.cpp

```diff
@@ -5648,6 +5718,69 @@ SARA_R5_error_t SARA_R5::getFileBlock(const String& filename, char* buffer, size
 
   return SARA_R5_ERROR_SUCCESS;
 }
+SARA_R5_error_t SARA_R5::getAvailableSize(size_t * size)
+{
+
+  // Get the remaining space available on the modem in bytes
+  char * command;
+  char * response;
+
+  command = sara_r5_calloc_char(strlen(SARA_R5_FILE_SYSTEM_LIST_FILES) + 3);
+  if (command==NULL)
+  {
+    return SARA_R5_ERROR_OUT_OF_MEMORY;
+  }
+
+  sprintf(command, "%s=1", SARA_R5_FILE_SYSTEM_LIST_FILES);
```

Comment:
Please add a comment here that AT+ULSTFILE=1 returns the Remaining free FS space expressed in bytes.
(I was worried it was doing a full LS - and would overflow the minimumResponseAllocation)